.lintstagedrc.json

```diff
@@ -1,5 +1,5 @@
 {
-  "*.ts": ["eslint --fix"],
+  "*.ts": ["eslint ./src --ext .ts --fix"],
```

Comment:
Do we not want to lint the `test` files as well?

Reply:
this is updated now